config/horizon.json

```diff
@@ -166,7 +166,8 @@
 		"woop": false,
 		"wordpress-action-search": false,
 		"wpcom-user-bootstrap": true,
-		"yolo/command-palette": true
+		"yolo/command-palette": true,
+		"woocommerce/core-profiler-passwordless-auth": true
```

Comment:
I'm curious if we intend to do user testing in horizon? AFAIK this is for public testing PCYsg-5bh-p2

Reply:
@ilyasfoo Good question. I think we should focus on internal testing first. It sounds like Horizon is intended for invite-only testing, but I believe Horizon works well for internal testing too. Staging is another option, though it's often busy with other ongoing tests

cc @adrianduffell